.github/actions/setup-google-cloud/action.yml

```
@@ -2,53 +2,33 @@ name: Setup Google Cloud SDK
 description: Prepare Google Cloud SDK in GitHub Actions
 
 inputs:
-  project-id:
-    description: Google Cloud SDK project id
-    default: exponentjs
-
-  project-zone:
-    description: Google Cloud SDK compute zone
-    default: us-central1
-
-  project-cluster:
-    description: Google Cloud SDK container cluster
-    default: general-central
-
-  service-key:
-    description: Google Cloud SDK service_account_key
-    default: ''
-
-  export-default-credentials:
-    type: boolean
+  auth:
     description: If it should export the default credentials
```

```suggestion
    description: If it should authenticate with Google Cloud and the Kubernetes cluster
```
Also, shouldn't this type be a boolean and look for true/false instead of empty strings?

---

That would be nice, but [action inputs do not have types](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs), they are all strings.  You can type the inputs to re-usable workflows, but not actions.